layers/gpu/cmd_validation/gpuav_draw.cpp

```diff
@@ -15,143 +15,77 @@
  * limitations under the License.
  */
 
+#include "gpu/cmd_validation/gpuav_draw.h"
+
 #include "gpu/core/gpuav.h"
 #include "gpu/cmd_validation/gpuav_cmd_validation_common.h"
 #include "gpu/error_message/gpuav_vuids.h"
+#include "gpu/resources/gpu_resources.h"
 #include "gpu/resources/gpuav_subclasses.h"
+#include "state_tracker/cmd_buffer_state.h"
 #include "state_tracker/render_pass_state.h"
 #include "gpu/shaders/gpu_error_header.h"
 #include "gpu/shaders/gpu_shaders_constants.h"
-#include "generated/cmd_validation_draw_vert.h"
-
-// See gpu/shaders/cmd_validation/draw.vert
-constexpr uint32_t kPushConstantDWords = 11u;
+#include "gpu/shaders/cmd_validation/draw_push_data.h"
+#include "generated/cmd_validation_draw_mesh_indirect_vert.h"
+#include "generated/cmd_validation_first_instance_vert.h"
+#include "generated/cmd_validation_count_buffer_vert.h"
+#include "generated/cmd_validation_draw_indexed_vert.h"
+#include "generated/cmd_validation_draw_indexed_indirect_index_buffer_vert.h"
+#include "generated/cmd_validation_draw_indexed_indirect_vertex_buffer_vert.h"
 
 namespace gpuav {
+namespace bufval {
```

Comment:
```suggestion
namespace cmdval {
```

I guess not sure what "buffer" refers to when literally everything is becoming a buffer now

Reply:
could also be 'drawval'

Reply:
actually why do we need `val`... everything is validation in GPU-AV (it is the `V` in it lol)

can this just be `gpuav::draw`?